### ISSUE TITLE
No SMART status for each individual drive

### ISSUE THREAD

**Issue by mr-each** [dak180/FreeNAS-Report#33]
Hello there!

Running TrueNAS-SCALE-23.10.2, RaidZ1 5 drives wide with only 4 drives connected, all Seagate HDDs.

I'm getting a proper zpool and SMARTs summary, zpool status reports for individual pool but no SMART statuses for individual drive. I can get desired output via `smartctl -H -A -l error /dev/DRIVE`. Have daily SMART tests configured, if it is related. Am I missing something, or SMART status for each drive should not be visible in e-mail by default? I've played with "includeSSD" and "includeSAS" options, but had no success.

**Comment by dak180** [comment]
Please attach a dumpfile (`report.sh -dc report.cfg`) so I can see what might be happening.

**Comment by mr-each** [comment]
Here you go [dumpfile.zip](https://github.com/dak180/FreeNAS-Report/files/14735117/dumpfile.zip)

**Comment by dak180** [comment]
@mr-each I do not see anything obvious at first glance; can you export the email as a pdf and attach it, please.

**Comment by mr-each** [comment]
Here's email as [PDF](https://github.com/dak180/FreeNAS-Report/files/14740350/email_export.pdf) and as [code](https://github.com/dak180/FreeNAS-Report/files/14740354/email_code.txt)

I've tried using [Bidule0hm](https://www.truenas.com/community/threads/scripts-to-report-smart-zpool-and-ups-status-hdd-cpu-t%C2%B0-hdd-identification-and-backup-the-config.27365/)'s [script](https://pastebin.com/9xBRFFuB) from TrueNAS forum - email formatting was screwed, but SMART statuses worked great

**Comment by dak180** [comment]
It seems to be skipping a for [code block](https://github.com/dak180/FreeNAS-Report/blob/master/report.sh#L2656) which, since it relies on the same array as the earlier HDD summary table does not make a lot of sense to me.  Can you do a manual run with `-x` set for the bash parent and post that output here (as a txt file) so I can see exactly what it is doing?

**Comment by mr-each** [comment]
OMG... I was trying to debug the script with the most stupid way I know - print smth in console, when I noticed that the code in my report.sh is not the same as in yours.

TL;DR I was using v1.8. Got the v1.8.1 and everything works now. I am sorry for being so dumb 😅
